eng/targets/ResolveReferences.targets

```diff
@@ -77,7 +78,8 @@
 
     <_UnusedProjectReferenceProvider Include="@(ProjectReferenceProvider)" Exclude="@(Reference)" />
 
-    <_CompilationOnlyReference Condition="'$(TargetFramework)' == 'netstandard2.0'" Include="@(Reference->WithMetadataValue('NuGetPackageId','NETStandard.Library'))" />
+    <_CompilationOnlyReference Condition="'$(TargetFramework)' == 'netstandard2.0'"
```

Comment:
Why move the conditions to a separate line for other items/properties, but keep the condition on the same line here?

Reply:
Mostly depended on line length and where my cursor was when I hit **Enter** 😈 I don't particularly care whether the `Include` or the `Condition` appears first or second. But, if you do, lemme know.